site/e2e/playwright.config.ts

```diff
@@ -147,7 +147,7 @@ export default defineConfig({
         gitAuth.validatePath,
       ),
       CODER_PPROF_ADDRESS: "127.0.0.1:" + coderdPProfPort,
-      CODER_EXPERIMENTS: e2eFakeExperiment1 + "," + e2eFakeExperiment2,
+      CODER_EXPERIMENTS: `multi-organization,${e2eFakeExperiment1},${e2eFakeExperiment2}`,
```

Comment:
Nit: Why do we need those `e2eFakeExperiment<number>`?

Reply:
I just work here 😅 they're already there, and I'm sure they're used in other tests